### PR TITLE
#360 fix TODO RemoteDaemon.GetChild

### PR DESCRIPTION
**Pull request by maxim-s**

### CODE REVIEW

File: src/core/Akka.Remote/RemoteSystemDaemon.cs [hunk 1]
```diff
@@ -276,28 +277,65 @@ private void HandleDaemonMsgCreate(DaemonMsgCreate message)
         public override IActorRef GetChild(IEnumerable<string> name)
         {
             string[] parts = name.ToArray();
-            //TODO: I have no clue what the scala version does
             if (!parts.Any())
                 return this;
 
-            string n = parts.First();
+            var n = parts.First();
             if (string.IsNullOrEmpty(n))
```
Comment (Silv3rcircl3) [inline src/core/Akka.Remote/RemoteSystemDaemon.cs]: Wouldn't it be easier to use FirstOrDefault?

**Comment by rogeralsing** [comment]
Looks good to me

File: src/core/Akka.Remote/RemoteSystemDaemon.cs (hunk 1 above)
Comment (maxim-s) [inline src/core/Akka.Remote/RemoteSystemDaemon.cs]: I'll check why l haven't done it, thank you

**Comment by maxim-s** [comment]
@rogeralsing @Silv3rcircl3 @sean-gilliam Updated

**Comment by Aaronontheweb** [comment]
Restarted the MNTR - going to see if I can get this in today before I finalize the binaries for 1.0.6

**Comment by Aaronontheweb** [comment]
@maxim-s could you rebase this on dev?

**Comment by Aaronontheweb** [comment]
@maxim-s sorry for the delay on this - re-running the MNTR now.

**Comment by maxim-s** [comment]
@Aaronontheweb updated

**Comment by Aaronontheweb** [comment]
@maxim-s argh, I'm sorry about this but it looks like there's a merge conflict again :\ - would you mind rebasing?

**Comment by Aaronontheweb** [comment]
Ah, nevermind - looks like that was a result of me tinkering around with protected branches :p

**Pull request merged**